src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/SourceWriter.cs

```diff
@@ -40,41 +37,39 @@ private void WriteInternal()
             WriteFileHeader();
 
             WriteLine("namespace Microsoft.CodeAnalysis.CSharp.Syntax.InternalSyntax");
-            WriteLine("{");
+            OpenBlock();
             WriteLine();
             this.WriteGreenTypes();
             this.WriteGreenVisitors();
             this.WriteGreenRewriter();
             this.WriteContextualGreenFactories();
             this.WriteStaticGreenFactories();
-            WriteLine("}");
+            CloseBlock();
```

Comment:
Functions that already exist and already emit a `{` and update the indentation for what is written.